pyuvdata/tests/test_uvdata.py

```diff
@@ -234,48 +227,46 @@ def teardown(self):
 
     def test_baseline_to_antnums(self):
         """Test baseline to antnum conversion for 256 & larger conventions."""
-        nt.assert_equal(self.uv_object.baseline_to_antnums(67585), (0, 0))
-        nt.assert_raises(
-            Exception, self.uv_object2.baseline_to_antnums, 67585)
+        assert self.uv_object.baseline_to_antnums(67585) == (0, 0)
+        pytest.raises(Exception, self.uv_object2.baseline_to_antnums, 67585)
 
         ant_pairs = [(10, 20), (280, 310)]
         for pair in ant_pairs:
             if np.max(np.array(pair)) < 255:
                 bl = self.uv_object.antnums_to_baseline(
                     pair[0], pair[1], attempt256=True)
                 ant_pair_out = self.uv_object.baseline_to_antnums(bl)
-                nt.assert_equal(pair, ant_pair_out)
+                assert pair == ant_pair_out
 
             bl = self.uv_object.antnums_to_baseline(
                 pair[0], pair[1], attempt256=False)
             ant_pair_out = self.uv_object.baseline_to_antnums(bl)
-            nt.assert_equal(pair, ant_pair_out)
+            assert pair == ant_pair_out
 
     def test_antnums_to_baselines(self):
         """Test antums to baseline conversion for 256 & larger conventions."""
-        nt.assert_equal(self.uv_object.antnums_to_baseline(0, 0), 67585)
-        nt.assert_equal(self.uv_object.antnums_to_baseline(257, 256), 594177)
-        nt.assert_equal(self.uv_object.baseline_to_antnums(594177), (257, 256))
+        assert self.uv_object.antnums_to_baseline(0, 0) == 67585
+        assert self.uv_object.antnums_to_baseline(257, 256) == 594177
+        assert self.uv_object.baseline_to_antnums(594177) == (257, 256)
         # Check attempt256
-        nt.assert_equal(self.uv_object.antnums_to_baseline(
-            0, 0, attempt256=True), 257)
-        nt.assert_equal(self.uv_object.antnums_to_baseline(257, 256), 594177)
+        assert self.uv_object.antnums_to_baseline(0, 0, attempt256=True) == 257
+        assert self.uv_object.antnums_to_baseline(257, 256) == 594177
         uvtest.checkWarnings(self.uv_object.antnums_to_baseline, [257, 256],
                              {'attempt256': True}, message='found > 256 antennas')
-        nt.assert_raises(
-            Exception, self.uv_object2.antnums_to_baseline, 0, 0)
+        pytest.raises(Exception, self.uv_object2.antnums_to_baseline, 0, 0)
```

Comment:
same question about the Exception type

Reply:
this tests teh following https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/63d6f7f7bb30ddea01f917b9a031a03ff52f20a5/pyuvdata/utils.py#L936-L939

Reply:
thanks!